### PR TITLE
cf: Move cf-kube-dist pipeline into the main CF pipeline

### PR DESCRIPTION
**Pull request by mook-as**
This means we only create the BOSH releases once, using a unified set of sources.

The helm section is to come.

### CODE REVIEW

File: cf-kube-dist/cf-kube-dist.yml [hunk 1]
```diff
@@ -1,70 +0,0 @@
...
-    access_key_id: {{aws-access-key}}
-    secret_access_key: {{aws-secret-key}}
-    bucket: {{s3-bucket}}
-    regexp: fissile/fissile-(.*)\.g.*linux.amd64\.tgz
-    private: true
-
-- name: s3.stampy-binary
-  type: s3
-  source:
-    access_key_id: {{s3-access-key}}
```
Comment (andreas-kupries) [inline cf-kube-dist/cf-kube-dist.yml]: `s3` --> `aws`, ditto fixes for secret-key, bucket, endpoint. See also s3.kube-dist.
Working form is seen with s3.certstrap-binary, and s3.fissile-binary above.

Comment (mook-as) [inline cf-kube-dist/cf-kube-dist.yml]: Addressed in 9561744baf162d8bc55f37494599457a01be52cb (I think)

Comment (andreas-kupries) [inline cf-kube-dist/cf-kube-dist.yml]: :eyes: and testing

Comment (andreas-kupries) [inline cf-kube-dist/cf-kube-dist.yml]: That looks ok.

File: cf/tasks/kube-dist.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+)
+(
+    # Workaround for dependencies on UAA that are silly
+    cd src
+    source .envrc
+    bin/settings/kube/ca.sh
+    head make/kube
+    make/kube
+    # XXX marky Helm is busted WRT to FISSILE_OUTPUT_DIR
+    /usr/bin/env FISSILE_OUTPUT_DIR="${PWD}/helm" make/kube helm
```
Comment (andreas-kupries) [inline cf/tasks/kube-dist.sh]: The helm files are generated. ☑️

File: cf/tasks/kube-dist.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    make/kube-dist
```
Comment (andreas-kupries) [inline cf/tasks/kube-dist.sh]: The helm files are **not** included in the zip file :-1:
This per the log of http://concourse.shorty.myen.van/teams/main/pipelines/marky-scf-develop/jobs/kube-dist/builds/46
and by looking at the zip in my `package-bits` trialing.

Comment (andreas-kupries) [inline cf/tasks/kube-dist.sh]: Ok. Issue is in SCF, not the pipeline and its tasks and jobs.

**Pull request merged**